web/src/components/molecules/Content/Form/fields/FieldTypesMap.ts

```diff
@@ -0,0 +1,17 @@
+import { TagField, DateField, BoolField, CheckboxField, URLField } from "./FieldComponents";
+import IntegerField from "./FieldComponents/IntegerField";
+import MarkdownField from "./FieldComponents/MarkdownField";
+import SelectField from "./FieldComponents/SelectField";
+import TextareaField from "./FieldComponents/TextareaField";
+
+export const FIELD_TYPE_COMPONENT_MAP = {
```

Comment:
I think it's better to export this in molecules/Content/Form/fields/FieldComponents/index.ts